docs/scripts/build-assembly.sh

```diff
@@ -31,3 +31,7 @@ do
     sed -i.bck 's|.. literalinclude:: ../../|.. literalinclude:: |g' $file
     rm $file.bck
 done
+
+# Remove 'self' from the master index
```

Comment:
What happens if we remove it directly on the `rst`? Not an important question really, just curiosity

Reply:
We get two entries in the menu on the `docs.daml.com` side as we have a link from their master table of contents to our master `index.rst` file.

From the `docs.daml.com` master index file:
```
.. toctree::
   :titlesonly:
   :maxdepth: 2
   :hidden:
   :caption: Create Daml Apps

   Daml Standard Library </index/create-daml-apps/standard-library/index>
   Integrate Daml with Off-Ledger Services </index/create-daml-apps/off-ledger/index>
   Developer Tools </index/create-daml-apps/developer-tools/index>
   Daml Finance </daml-finance/index>
```